### ISSUE TITLE
Api version deprecated

### ISSUE THREAD

**Issue by ghost** [jleetutorial/kubernetes-demo#40]
Change apps/v1beta2 to apps/v1

**Comment by LugarJE** [comment]
yes, this was quite helpful for latter versions. Kubernetes v1.22.2 and minikube v1.23.2 on Debian 10.10. Just beware of the Audits Flag was deprecated and is broken in these later versions +1.19. See later issue for details.
